playground/ParameterEndToEnd/ParameterEndToEnd.AppHost/Program.cs

```diff
@@ -17,11 +15,10 @@
 // The expression below is a bit more complex than the average developer app would
 // probably have, but in our repo we'll probably want to experiment with seperately
 // deployed resources a little bit.
-var simulateProduction = builder.Configuration.GetValue<bool>("SimulateProduction", false);
 var db = builder.Environment.EnvironmentName switch
 {
     "Production" => builder.AddConnectionString("db"),
-    _ => simulateProduction ? builder.AddConnectionString("db") : builder.AddSqlServer("sql").AddDatabase("db")
+    _ => builder.AddSqlServer("sql").PublishAsConnectionString().AddDatabase("db")
```

Comment:
Do we still need the `"Production"` switch here? Why can't this just be:

```
var db = builder.AddSqlServer("sql").PublishAsConnectionString().AddDatabase("db")
```
?

Reply:
We don't, I just left in this crazy code 😄